src/utils.ts

```diff
@@ -24,10 +24,10 @@ export const formatDuration = (duration: number): string => {
 };
 
 // Auxillary test data functions
-export const getFullTestName = (testResult: AssertionResult): string =>
-  testResult.ancestorTitles && testResult.ancestorTitles.length
-    ? `${testResult.ancestorTitles.join(" / ")} / ${testResult.title}`
-    : testResult.title;
+export const getFullTestName = (ancestorTitles: string[], testResultTitle: string): string =>
```

Comment:
Can we use the single `testResult` parameter like before? When you call this, you are passing two properties of the `testResult` so why not pass the whole thing?